cfy_manager/networks/networks.py

```diff
@@ -20,17 +20,30 @@
 from ..logger import get_logger, setup_console_logger
 from ..constants import (
     NETWORKS_DIR,
+    CLOUDIFY_HOME_DIR,
+    VERBOSE_HELP_MSG,
 )
 from ..utils.certificates import (
     create_internal_certs,
     load_cert_metadata,
     store_cert_metadata
 )
+from ..config import config
+from ..components.service_names import MANAGER
+from ..components.components_constants import HOSTNAME
+from ..components.globals import _set_hostname
 from ..utils.scripts import run_script_on_manager_venv
 
 SCRIPT_DIR = join(NETWORKS_DIR, 'scripts')
 REST_HOME_DIR = '/opt/manager'
 
+CONFIG_FILE_HELP_MSG = (
```

Comment:
We cannot import it from here https://github.com/cloudify-cosmo/cloudify-manager-install/blob/master/cfy_manager/main.py#L163? 
If it creates a cyclic dependency issue, you might want to take it out to a different file.

Reply:
This is not imported from main module, I just re-define it on the networks module again since I cannot import it from main

Reply:
If this can be imported from main.py then it should be moved to a different file that is accessible both from here and from main.py. Changing this help message in the future would need to be done in one place only.

Reply:
will move this to one place and make it accessible for both modules